src/tests/ftest/util/spdk.py

```diff
@@ -86,25 +86,25 @@ def nvme_cleanup_thread(result_queue, hostname, debug=True):
     result_queue.put("PASS")
     host.disconnect()
 
-def nvme(hostlist, operation="cleanup"):
+def nvme_main(hostlist, operation=False):
```

Comment:
It's typical to indicate _internal-use_ functions by prefixing their names with an `_`.